cobra/evaluation/pigs_tables.py

```diff
@@ -170,7 +170,7 @@ def plot_incidence(pig_tables: pd.DataFrame,
         ax.set_xlabel('{} bins' ''.format(variable), fontsize=16)
         ax.xaxis.set_tick_params(labelsize=14)
         plt.setp(ax.get_xticklabels(),
-                 rotation=45, ha="right", rotation_mode="anchor")
+                 rotation=90, ha="right", rotation_mode="anchor")
```

Comment:
Doesn't that make it less readable? Is it because for a dataset you were evaluating, 45° angle didn't display nice for some reason - even longer bin labels than I had, for example? + when rotation=90, ha="center" might be better.

Reply:
I reconsidered, and put it back to `rotation=45`. 😀 I had indeed one plot with a very large number of labels, and vertical rotation made it more or less readable. But agree to keep it like this, as it makes the typical cases optimally readable.

Reply:
Okidoki!